src/components/calcite-shell/usage/advanced.md

```diff
@@ -0,0 +1,115 @@
+#### Advanced
+
+Renders a shell with leading and trailing floating panels, action bar/pad, block, flow, tip manager, footer
+
+```html
+<calcite-shell>
+  <calcite-shell-panel slot="primary-panel" layout="leading" detached>
+    <calcite-action-pad id="action-pad" placement="side" slot="action-pad" hidden>
+      <calcite-action-group>
+        <calcite-action text="Add">
+          <!-- icon -->
```

Comment:
Maybe we can use `<calcite-icon>` here now that we have that component to use? Would apply to other usage files so this can be a follow up issue to address.